fontforgeexe/bitmapview.c

```diff
@@ -762,19 +764,23 @@ static void BVExpose(BitmapView *bv, GWindow pixmap, GEvent *event ) {
 		? (bv->bc->byte_data ? 0x008800 : 0x004400 )
 		: 0x00ff00;
 	memset(&cvtemp,'\0',sizeof(cvtemp));
+	memset(&cvtabtemp,'\0',sizeof(cvtabtemp));
 	cvtemp.v = bv->v;
 	cvtemp.width = bv->width;
 	cvtemp.height = bv->height;
-	cvtemp.scale = bv->scscale*bv->scale;
-	cvtemp.xoff = bv->xoff/* *bv->scscale*/;
-	cvtemp.yoff = bv->yoff/* *bv->scscale*/;
+
+	cvtabtemp.scale = bv->scscale*bv->scale;
+	cvtabtemp.xoff = bv->xoff/* *bv->scscale*/;
+	cvtabtemp.yoff = bv->yoff/* *bv->scscale*/;
+	cvtemp.cvtabs[0] = cvtabtemp;
```

Comment:
Pretty sure you don't need this `cvtabtemp` any more, you can just assign directly to cvtemp.cvtabs[0]